src/services/item/plugins/file/test/index.test.ts

```diff
@@ -600,16 +600,11 @@ describe('File Item routes tests', () => {
           setTimeout(async () => {
             await expect(copyObjectMock).toHaveBeenCalled();
 
-            const items1 = await testUtils.rawItemRepository.find({ where: { name: item.name } });
-            expect(items1).toHaveLength(1);
+            const items = await testUtils.rawItemRepository.find({ where: { name: item.name } });
+            expect(items).toHaveLength(2);
```

Comment:
Just to be sure here, when you copy an item, the copy has exactly the same name ?

Reply:
It depends on the destination of the copy
From this diagram :
```mermaid
flowchart TD
    A-->B
    A-->C
```

1. If we copy `C` in `B`. The copy will have the exact same name
```mermaid
flowchart TD
    A-->B
    A-->C((C))
    B-->copy((C))
```

2. If we copy `C` in `A`. The copy will have a suffix, because there is already an Item named `C` in `A`
```mermaid
flowchart TD
    A-->B
    A-->C((C))
    A-->copy(("C (2)"))
```

Probably we need to confirm this is the behavior we want before merging ? @spaenleh  @pyphilia

Reply:
Ah ok thanks for your explanations and diagrams, it clearer to me ^^

Reply:
it is important to know that a copied item is a copy ? If it is, should we include "copy" in the new name ?